module/classes/DevModePerformance.mjs

```diff
@@ -17,14 +17,17 @@ export class DevModePerformance {
     });
   }
 
-  static actorCRUDTest = async (iterations = 1000) => {
-    console.log('running test with iterations:', `${iterations}`, !!iterations, iterations || 1000);
+  static actorCRUDTest = async (type, iterations = 1000) => {
```

Comment:
```suggestion
  static actorCRUDTest = async ({ type, iterations = 1000} = {}) => {
```
by feeding it the object output from `FormDataExtended`, this will need to destructure that object

Reply:
I thought I did that already... I honestly had problems with git that might've lost that.